docusaurus/docs/how-to-guides/data-source-plugins/profile-backend-plugin.md

```diff
@@ -13,14 +13,17 @@ keywords:
   - back-end
 ---
 
-This guide provides instructions for configuring a backend plugin to enable certain diagnostics when it starts, generating _profiling data_. Profiling data provides potentially useful information
+This guide provides instructions for configuring a plugin's backend to enable certain diagnostics when it starts, generating _profiling data_. Profiling data provides potentially useful information
 for investigating certain performance problems, such as high CPU or memory usage, or when you want to use [continuous profiling](https://grafana.com/oss/pyroscope/).
 
 ## Configure profiling data
 
 The [Grafana configuration file](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/) allows you to configure profiling under the `[plugin.<plugin ID>]`.
 
-In this section of the file, specify the `<plugin ID>`, a unique identifier, for the backend plugin you want to profile, for example, [grafana-github-datasource](https://grafana.com/grafana/plugins/grafana-github-datasource/), together with the profiling configuration options (detailed in sub-sections below).
+In this section of the file, specify: 
+
+- The `<plugin ID>`, a unique identifier, for the backend you want to profile. For example, [grafana-github-datasource](https://grafana.com/grafana/plugins/grafana-github-datasource/). 
```

Comment:
I think we can omit here we are talking about plugins with a backend since we have specified that before.

```suggestion
- The `<plugin ID>`, a unique identifier, for the plugin you want to profile. For example, [grafana-github-datasource](https://grafana.com/grafana/plugins/grafana-github-datasource/). 
```